chapters/publication.tex

```diff
@@ -1,96 +1,178 @@
+particularly if you are new to working with plain text code and file management.
+It is also unfortunately weak with spelling and grammar checking.
+This is because \LaTeX\ requires that all formatting be done in its special code language,
 and it is not particularly informative when you do something wrong.
 This can be off-putting very quickly for people
-who simply want to get to writing, like lead authors.
-Therefore, if we want to take advantage of the features of \LaTeX,
-without getting stuck in the weeds of it,
-we will need to adopt a few tools and tricks to make it effective.
-
-The first is choice of software. The easiest way
-for someone new to \LaTeX\ to be able to ``just write''
-is often the web-based Overleaf suite.\sidenote{\url{https://www.overleaf.com}}
-Overleaf offers a \textbf{rich text editor}
-that behaves pretty similarly to familiar tools like Word. 
-With minimal workflow adjustments, you can
-to show coauthors how to write and edit in Overleaf,
-so long as you make sure you are always available to troubleshoot
-\LaTeX\ crashes and errors. It also offers a convenient selection of templates
-so it is easy to start up a project
-and replicate a lot of the underlying setup code.
-One  of the most common issues you will face while using Overleaf will be special characters, namely
-\texttt{\&}, \texttt{\%}, and \texttt{\_},
-which need to be \textbf{escaped} (instructed to interpret literally)
-by writing a backslash (\texttt{\textbackslash}) before them,
-such as  \texttt{40\textbackslash\%} for the percent sign to function.
-Another issue is that you need to upload input files
-(such as figures and tables) manually.
-This can create conflicts when these inputs are still being updated --
-namely, the main document not having the latest results.
-One solution is to move to Overleaf only once there will not be substantive changes to results.
-
-Other popular desktop-based tools for writing \LaTeX are TeXstudio\sidenote{\url{https://www.texstudio.org}} and atom-latex\sidenote{\url{https://atom.io/packages/atom-latex}}.
-They allow more advanced integration with Git,
-among other advantages, but the entire team needs to be comfortable
+who simply want to get to writing, like senior researchers.
+While integrated editing and compiling tools like TeXStudio\sidenote{
+  \url{https://www.texstudio.org}}
+and \texttt{atom-latex}\sidenote{
+  \url{https://atom.io/packages/atom-latex}}
+offer the most flexibility to work with \LaTeX\ on your computer,
+such as advanced integration with Git,
+the entire group of writers needs to be comfortable
 with \LaTeX\ before adopting one of these tools.
+They can require a lot of troubleshooting at a basic level at first,
+and non-technical staff may not be willing or able to acquire the required knowledge.
+Therefore, to take advantage of the features of \LaTeX,
+while making it easy and accessible to the entire writing team,
+we need to abstract away from the technical details where possible.
```

Comment:
This feels like it should be the intro to the latex in the cloud section - it's a surprising jump when the next paragraph is bibtex

Reply:
e3fdde023cc61fb4d00b8e2b7a85ef8a66430d27